src/components/StaffGrid/index.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const StaffGrid = ({ staff }) => (
+  <div className="staff-grid row">
+    {staff.map(staffer => (
+      <div className="staffer-wrapper col-6 col-md-4 col-lg-3" key={staffer.uuid}>
+        <img src={staffer.profile_image_url} className="rounded-circle w-50" alt="" />
+        <div className="d-flex flex-column staffer-details">
+          <span className="name font-weight-bold">{staffer.given_name} {staffer.family_name}</span>
+          {staffer.position &&
+            <React.Fragment>
+              <div className="title font-italic">{staffer.position.title}</div>
+              {staffer.position.organization_name && <div className="organization">{staffer.position.organization_name}</div>}
+            </React.Fragment>
+          }
+        </div>
+      </div>
+    ))}
+  </div>
+);
+
+StaffGrid.propTypes = {
```

Comment:
The prop types here are definitely excessive, I was just trying to capture all the possible data from the API, in case we do need any of this information elsewhere. Additionally, I tried to capture what seemed like required fields based off discovery admin, but I might be wrong

Reply:
Yeah, I usually just leave it at PropTypes.shape(), but I know there are good arguments for being more explicit.